mysql-test/suite/innodb/t/log_encrypt_kill2.test

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE tne_1(c1 INT, c2 varchar(2000)) ENGINE = InnoDB;
+INSERT INTO tne_1 VALUES (1,REPEAT('a',1990)),(2,REPEAT('b',1990)),(100,REPEAT('c',1990));
+SELECT c1,LEFT(c2,10) FROM tne_1;
+--source include/kill_mysqld.inc
```

Comment:
a checkpoint  before kill doesn't guarantee recovery.
You need to disable log_checkpoint/ checkpointer threads
may be page cleaner threads too. Check other tests